spatial/r3/vector.go

```diff
@@ -17,6 +17,15 @@ func (p Vec) Add(q Vec) Vec {
 	return p
 }
 
+// Cross returns the vector cross product of p and q.
```

Comment:
we could perhaps replace `and` with the cross symbol (a simple `x` would do, but we could also reach for UTF8's `×`)

Reply:
The UTF-8 symbol is what we usually use where we use a symbol, though we often don't use a symbol.